src/domain/collaboration/aspect/aspect.service.ts

```diff
@@ -132,7 +132,7 @@ export class AspectService {
 
   public async getCardProfile(aspect: IAspect): Promise<ICardProfile> {
     const aspectLoaded = await this.getAspectOrFail(aspect.id, {
-      relations: ['profile'],
+      relations: ['profile', 'profile.tagset'],
```

Comment:
the proper way to do this imo is to provide either options or the relation param - the method is loading the profile from a card, the tagset is used just in one use-case where the method is used, the other use-cases probably won't need the tagset loaded here.